app/services/idv/steps/inherited_proofing/agreement_step.rb

```diff
@@ -0,0 +1,20 @@
+module Idv
+  module Steps
+    module InheritedProofing
+      class AgreementStep < InheritedProofingBaseStep
+        STEP_INDICATOR_STEP = :getting_started
```

Comment:
:getting_started or :agreement? `InheritedProofing::GettingStartedStep:: STEP_INDICATOR_STEP` == :getting_started

Reply:
We are good on this, step_indicator_step is used to indicate what step on UI (blue breadcrumbs that show porgress) we are on. This is as per Figma, https://www.figma.com/file/WOM0MdIaNCJdcj6KCSLvM3/VAIP-flows%2C-including-errors-and-failures-%5BLG-6186%5D?node-id=996%3A3242

Reply:
Ah ty for the clarification!